environments/india/terraform.yml

```diff
@@ -76,7 +76,7 @@ servers:
     os: jammy
 
   - server_name: "celery8-india"
-    server_instance_type: "t3.xlarge"
+    server_instance_type: "t3a.large"
```

Comment:
I see both the class and the size changed. Intentional or unintentional?

Reply:
Intentional as average utilization has been 30% for celery servers.